Library/Formula/vdirsyncer.rb

```diff
@@ -12,13 +12,13 @@ class Vdirsyncer < Formula
     sha256 "d0ad5bd159acdfeaf28aec0abda9f34d35750c4377624a365180c24798a8059c" => :mavericks
   end
 
-  option "without-keyring", "Build without python-keyring support"
+  option "with-remotestorage", "Build with support for remote-storage"
```

Comment:
If it's just a resource rather than additionally compiled dependency: I think it's probably worth just being default?